docs/architecture/adr/0021-console-logging.md

```diff
@@ -0,0 +1,114 @@
+---
+adr: "0021"
+status: In progress
+date: 2023-07-13
+tags: [server]
+---
+
+# 0021 - Logging with the Console
+
+<AdrTable frontMatter={frontMatter}></AdrTable>
+
+## Context and Problem Statement
+
+As the server platform has matured so have the various logging extensions to support additional use
+cases and customer requests. [Serilog][serilog] is in place via shared logic for all services,
+initiated at startup, and over time additional its "sinks" with their needed configuration have been
+added as dependencies leaving this core logging layer with a growing list of conditions on how and
+when to use certain types of structured logging. Maintenance needs have grown to keep sink
+dependencies up to date and more are desired to be added. Some of the presently-available sinks have
+very little use and / or better alternatives now exist.
```

Comment:
I think this section is a bit hard to follow.

Thoughts on splitting it into two paragraphs, starting with how the current logging solution works, and ending with why we want to change it. Something like.

> Bitwarden uses Serilog with both standard and custom logic to handle multiple outputs ....
> 
> Maintaining sinks that are never used is annoying .... and custom logic for file rotation ...